blogposts/2021/better-onboarding-advice-from-3-engineering-leaders.md

```diff
@@ -0,0 +1,125 @@
+She's now the director of the mentorship program at PowerToFly, a non-profit dedicated to helping women grow their careers.
+
+Ryan Djurovich is a DevOps Manager at Xero, Australia's largest accounting software company. 
+He leads the team responsible for developer infrastructure, experience, and tooling.
+
+Jean du Plessis is a Director of Engineering at Sourcegraph, where he leads the Developer Insights organization, which includes the frontend platform, extensibility, code insights, and API docs teams.
+
+The 3 offered their thoughts on the process of onboarding new engineers to the team, revealing some strong parallels as well as some particular areas of emphasis and focus.
+
+## Help new hires see the bigger picture with multi-tiered onboarding
+
+Multi-tiered onboarding provides a layered, holistic view of the organization.
+
+All 3 engineering leaders described a multi-tier onboarding process with steps both common across the company and organization and ones specific to the new hire's immediate team. 
+Some things should be standard across the organization: setting up and familiarizing oneself with shared collaboration tools (email, the code host, chat, CI/CD, the code review) and understanding the "big picture" of what the company does and how the various organizations interact with each other. 
+Department heads can give regular presentations to new hires about what their organization does. 
+
+Individual teams should also think about how to onboard new members to the things that are specific to the team, such as setting up the development environment that members of the team use, meeting and establishing relationships with teammates, and understanding what’s necessary to ship that first bug fix or feature enhancement.
+
+## Share high-level context
+
+High-level context is the hardest to transfer. It’s the hardest to document (without going out of date) and you can't automate it, because this really just reduces to grokking the overall architecture, which isn’t the same as truly understanding how the pieces fit together and knowing what the core assumptions and tension points are. 
+However, it's important to transfer knowledge, including historical knowledge, otherwise you can get into a position where all the members of the team who feel confident making big changes, not just small, incremental ones, are gone. 
+
+Jean shared: “This exact scenario played out for a new engineer recently. They love context and understanding the history behind decisions. 
+Thankfully, Sourcegraph has this practice of putting [Requests for Comments (RFCs)](https://about.sourcegraph.com/handbook/communication/rfcs) together when we want to introduce any changes or make architectural decisions. 
+We were able to refer to this document, which even predates my time, that gave context on these decisions. Because even the people who worked on it at the time eventually forget about it.” 
```

Comment:
```suggestion
We were able to refer to this document, which even predates my time, that gave context on these decisions. Because even the people who worked on it at the time eventually forget about it." 
```